metrics/storage/webtooling.sh

```diff
@@ -25,7 +25,17 @@ configuration_file="/usr/share/defaults/kata-containers/configuration.toml"
 TESTDIR="${TESTDIR:-/testdir}"
 file_path="/web-tooling-benchmark"
 file_name="output"
-CMD="mkdir -p ${TESTDIR}; cd $file_path && node dist/cli.js > $file_name"
+
+# Options to control the start of the workload using a trigger-file
+dst_dir="/host"
+src_dir="/tmp"
+trigger_file="$RANDOM.txt"
+guest_trigger_file="$dst_dir/$trigger_file"
+host_trigger_file="$src_dir/$trigger_file"
+start_script="webtooling_start.sh"
+
+CMD="$dst_dir/$start_script"
+MOUNT_OPTIONS="type=bind,src=$src_dir,dst=$dst_dir,options=rbind:ro"
```

Comment:
so, you are passing all the tmpfs (/tmp) to the container? that's not cool, can you create a temporal directory instead `mktemp -d`?